src/entry.js

```diff
@@ -46,7 +46,7 @@ module.exports = (argv) => {
   // create the context, logs thread through this
   const context = utils.createContext({command, args, argOpts});
 
-  if (argOpts.version || argOpts.v) {
+  if (command === 'help' && (argOpts.version || argOpts.v)) {
```

Comment:
Ah, this was added because in other commands (e.g. `zapier delete --version` would have exited early. 

`zapier --version` is not gone, though. Could we instead do `!command` or similar to try to keep the `zapier --version` working?